ci-operator/step-registry/openstack/scale/masters/openstack-scale-masters-commands.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+	# shellcheck disable=SC1090
+	source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+if [ -z "$SCALE_MASTERS_FLAVOR_ALTERNATE" ]; then
+	info "All defined environment variables for the step are empty. Exiting."
+	exit 0
+fi
+
+declare patch=''
+
+function info() {
+	printf '%s: %s\n' "$(date --utc +%Y-%m-%dT%H:%M:%SZ)" "$*"
+}
+
+function wait_for_cpms_done() {
+  if
+    ! oc wait --timeout=90m --for=condition=Progressing=false controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.spec.replicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.updatedReplicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.replicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null || \
+    ! oc wait --timeout=90m --for=jsonpath='{.status.readyReplicas}'=3 controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster 1>/dev/null; then
+      info "CPMS not scaled to 3 replicas"
+      oc get controlplanemachineset.machine.openshift.io -n openshift-machine-api cluster
```

Comment:
You may want to add `set +x` before the `oc` command so that we'll get the output of the commands. Because now, the `oc get` and `oc describe` do not serve any purpose since their output is lost.